ory-tunnel: serviceName needs to be specified for statefulSet

**Describe the bug**
error: error validating "./openline-setup/deployment/infra/k8s/ory-tunnel.yaml": error validating data: ValidationError(StatefulSet.spec): missing required field "serviceName" in io.k8s.api.apps.v1.StatefulSetSpec; if you choose to ignore these errors, turn validation off with --validate=false
[ERROR] error: error validating "./openline-setup/deployment/infra/k8s/ory-tunnel.yaml": error validating data: ValidationError(StatefulSet.spec): missing required field "serviceName" in io.k8s.api.apps.v1.StatefulSetSpec; if you choose to ignore these errors, turn validation off with --validate=false

**How to reproduce**
openline dev ory tunnel

--- a/deployment/infra/k8s/ory-tunnel.yaml
+++ b/deployment/infra/k8s/ory-tunnel.yaml
@@ -7,6 +7,7 @@ spec:
   selector:
     matchLabels:
       app: ory-tunnel        # This deployment applies to any Pods matching the specified label
+  serviceName: ory-tunnel
   template:                      # This deployment will create a set of pods using the configurations in this template
     metadata:
       labels:                    # The labels that will be applied to all of the pods in this deployment

